service/src/main/kotlin/app/cash/backfila/ui/components/BackfillsTable.kt

```diff
@@ -0,0 +1,73 @@
+package app.cash.backfila.ui.components
+
+import app.cash.backfila.dashboard.UiBackfillRun
+import kotlinx.html.TagConsumer
+import kotlinx.html.div
+import kotlinx.html.h1
+import kotlinx.html.table
+import kotlinx.html.tbody
+import kotlinx.html.td
+import kotlinx.html.th
+import kotlinx.html.thead
+import kotlinx.html.tr
+
+fun TagConsumer<*>.BackfillsTable(running: Boolean, backfills: List<UiBackfillRun>) {
+  val title = if (running) "Running" else "Paused"
+
+  div("px-4 sm:px-6 lg:px-8 py-5") {
+    div("sm:flex sm:items-center") {
+      div("sm:flex-auto") {
+        h1("text-base font-semibold leading-6 text-gray-900") { +"""Backfills ($title)""" }
+        // p("mt-2 text-sm text-gray-700") { +"""A list of all the users in your account including their name, title, email and role.""" }
+      }
+    }
+    div("mt-8 flow-root") {
+      div("-mx-4 -my-2 overflow-x-auto sm:-mx-6 lg:-mx-8") {
```

Comment:
This is still a bit cryptic for us service devs :P

Reply:
https://tailwindui.com/ and https://tailwindcss.com/docs/installation are your friends